src/components/dapps/ModalCreateDapps.vue

```diff
@@ -471,8 +325,8 @@ export default defineComponent({
 
       const abiData = abi.value as Abi | AnyJson;
 
-      console.log('s', abiData);
-      console.log('w', wasm.value);
+      // console.log('s', abiData);
+      // console.log('w', wasm.value);
 
       let uploadTx: SubmittableExtrinsic<'promise'> | null = null;
```

Comment:
I get an error when I'm trying to deploy a contract to the local development network.

![bandicam 2021-07-06 14-03-57-558](https://user-images.githubusercontent.com/40356749/124546682-fc3f4480-de65-11eb-8ac5-7999854f4676.jpg)
```
ModalCreateDapps.vue?77fa:233 _onFailed CodeSubmittableResult {dispatchError: Type, dispatchInfo: Map(3), internalError: undefined, events: Array(1), status: Type, …}blueprint: undefinedcontract: undefineddispatchError: Type {registry: TypeRegistry, createdAtHash: undefined, …}dispatchInfo: Map(3)[[Entries]]0: {"weight" => Type}1: {"class" => Type}2: {"paysFee" => Type}class: (...)createdAtHash: undefinedpaysFee: (...)registry: TypeRegistry {createdAtHash: undefined, __private_69_classes: Map(216), __private_70_definitions: Map(849), __private_71_metadataCalls: {…}, __private_72_metadataErrors: {…}, …}weight: (...)Type: (...)defKeys: (...)encodedLength: (...)hash: (...)isEmpty: (...)size: (...)__private_44_jsonMap: Map(0) {}__private_45_Types: {weight: ƒ, class: ƒ, paysFee: ƒ}get class: () => this.get(key)get paysFee: () => this.get(key)get weight: () => this.get(key)__proto__: Mapevents: [Map(3)]0: Map(3) {"phase" => Type, "event" => Map(2), "topics" => Array(0), …}length: 1__proto__: Array(0)internalError: undefinedstatus: Type {registry: TypeRegistry, createdAtHash: undefined, …}isCompleted: (...)isError: (...)isFinalized: (...)isInBlock: (...)isWarning: (...)__proto__: SubmittableResult
_onFailed @ ModalCreateDapps.vue?77fa:233
eval @ util.ts?cb96:119
eval @ util.ts?cb96:116
eval @ index.js?d120:74
setTimeout (async)
eval @ index.js?d120:74
SafeSubscriber.__tryOrUnsub @ Subscriber.js?1453:192
SafeSubscriber.next @ Subscriber.js?1453:130
Subscriber._next @ Subscriber.js?1453:76
Subscriber.next @ Subscriber.js?1453:53
TapSubscriber._next @ tap.js?c4cc:51
Subscriber.next @ Subscriber.js?1453:53
Subscriber._next @ Subscriber.js?1453:76
Subscriber.next @ Subscriber.js?1453:53
TapSubscriber._next @ tap.js?c4cc:51
Subscriber.next @ Subscriber.js?1453:53
SwitchMapSubscriber.notifyNext @ switchMap.js?d792:71
SimpleInnerSubscriber._next @ innerSubscribe.js?cf86:14
Subscriber.next @ Subscriber.js?1453:53
Subscriber._next @ Subscriber.js?1453:76
Subscriber.next @ Subscriber.js?1453:53
MapSubscriber._next @ map.js?ebb6:41
Subscriber.next @ Subscriber.js?1453:53
Subject.next @ Subject.js?2bd2:47
ReplaySubject.nextInfiniteTimeWindow @ ReplaySubject.js?4b96:41
Subscriber._next @ Subscriber.js?1453:76
Subscriber.next @ Subscriber.js?1453:53
DistinctUntilChangedSubscriber._next @ distinctUntilChanged.js?69dd:55
Subscriber.next @ Subscriber.js?1453:53
Subscriber._next @ Subscriber.js?1453:76
Subscriber.next @ Subscriber.js?1453:53
MapSubscriber._next @ map.js?ebb6:41
Subscriber.next @ Subscriber.js?1453:53
CombineLatestSubscriber.notifyNext @ combineLatest.js?7738:83
InnerSubscriber._next @ InnerSubscriber.js?acf8:15
Subscriber.next @ Subscriber.js?1453:53
Subject.next @ Subject.js?2bd2:47
ReplaySubject.nextInfiniteTimeWindow @ ReplaySubject.js?4b96:41
Subscriber._next @ Subscriber.js?1453:76
Subscriber.next @ Subscriber.js?1453:53
eval @ index.js?33d6:230
Promise.then (async)
eval @ index.js?33d6:229
Observable._trySubscribe @ Observable.js?e9b9:43
Observable.subscribe @ Observable.js?e9b9:29
ConnectableObservable.connect @ ConnectableObservable.js?2a11:34
eval @ refCountDelay.js?0d85:17
Observable._trySubscribe @ Observable.js?e9b9:43
Observable.subscribe @ Observable.js?e9b9:29
subscribeToResult @ subscribeToResult.js?ce8b:13
CombineLatestSubscriber._complete @ combineLatest.js?7738:62
Subscriber.complete @ Subscriber.js?1453:65
eval @ subscribeToArray.js?fad2:7
Observable._trySubscribe @ Observable.js?e9b9:43
Observable.subscribe @ Observable.js?e9b9:29
CombineLatestOperator.call @ combineLatest.js?7738:32
Observable.subscribe @ Observable.js?e9b9:24
MapOperator.call @ map.js?ebb6:18
Observable.subscribe @ Observable.js?e9b9:24
eval @ memo.js?4a79:13
Observable._trySubscribe @ Observable.js?e9b9:43
Observable.subscribe @ Observable.js?e9b9:29
CatchOperator.call @ catchError.js?f59d:16
Observable.subscribe @ Observable.js?e9b9:24
DistinctUntilChangedOperator.call @ distinctUntilChanged.js?69dd:13
Observable.subscribe @ Observable.js?e9b9:24
ConnectableObservable.connect @ ConnectableObservable.js?2a11:34
eval @ refCountDelay.js?0d85:17
Observable._trySubscribe @ Observable.js?e9b9:43
Observable.subscribe @ Observable.js?e9b9:29
MapOperator.call @ map.js?ebb6:18
Observable.subscribe @ Observable.js?e9b9:24
CatchOperator.call @ catchError.js?f59d:16
Observable.subscribe @ Observable.js?e9b9:24
innerSubscribe @ innerSubscribe.js?cf86:89
SwitchMapSubscriber._innerSub @ switchMap.js?d792:49
SwitchMapSubscriber._next @ switchMap.js?d792:39
Subscriber.next @ Subscriber.js?1453:53
Subject.next @ Subject.js?2bd2:47
ReplaySubject.nextInfiniteTimeWindow @ ReplaySubject.js?4b96:41
Subscriber._next @ Subscriber.js?1453:76
Subscriber.next @ Subscriber.js?1453:53
DistinctUntilChangedSubscriber._next @ distinctUntilChanged.js?69dd:55
Subscriber.next @ Subscriber.js?1453:53
Subscriber._next @ Subscriber.js?1453:76
Subscriber.next @ Subscriber.js?1453:53
update @ index.js?33d6:298
value @ index.js?ec14:270
value @ index.js?ec14:205
Show 57 more frames
```

Reply:
@hoonsubin  can you upload `erc721.contract` file? and it seems deployment fails when the endowment is less than the value that the contract requires. when I try to small amount of endowment, it fails as same above. and this situation has already been shared before. so, what do you think the good method to resolve this? 
e.g, in polkadot explorer, show the fee like this.
<img width="429" alt="스크린샷 2021-07-06 오후 3 28 14" src="https://user-images.githubusercontent.com/1606820/124553028-34975080-de6f-11eb-8f6d-84c6e642d7ae.png">

if I set less the fee than above, the transaction goes fail. so, to improve this, there's way to calculate and set fee automatically.

Reply:
it seems not exactly above that unit, the following error occurs below about 7 kilo. (flipper.contract). Do you know the conditions for this?
@hoonsubin 
![스크린샷 2021-07-07 오후 8 25 02](https://user-images.githubusercontent.com/1606820/124751730-f169da00-df61-11eb-936e-83cfb7dd0b88.png)

Reply:
[erc20.zip](https://github.com/PlasmNetwork/plasm-apps-vue/files/6780964/erc20.zip)
Here is the compiled contract file. I believe this topic was discussed on Slack right?

Reply:
not really. it's related with ux issue. we should decide if throw an error after sending or show estimated fee in advance. maybe it should be separate issue.

and when I tried to deploy `erc20` contract, it works well.
![스크린샷 2021-07-08 오후 2 11 39](https://user-images.githubusercontent.com/3349973/124866436-63d8c980-dff7-11eb-8eee-57738849dcbe.png)
can you trying again?

Reply:
not really. it's related with ux issue. we should decide if throw an error after sending or show estimated fee in advance. maybe it should be separate issue.

and when I tried to deploy erc20 contract, it works well.
can you try again?

Reply:
[erc721.zip](https://github.com/PlasmNetwork/plasm-apps-vue/files/6788858/erc721.zip)
when I try to upload this smart contract on my local development node, I get the following error during the upload process.
![bandicam 2021-07-09 09-06-40-676](https://user-images.githubusercontent.com/40356749/125026266-3f332f00-e095-11eb-96d4-a7f02cd0034f.jpg)

```
ModalCreateDapps.vue?77fa:318 Uncaught TypeError: Cannot read property 'value' of undefined
    at eval (webpack-internal:///./node_modules/cache-loader/dist/cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/@vue/cli-plugin-typescript/node_modules/ts-loader/index.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader-v16/dist/index.js?!./src/components/dapps/ModalCreateDapps.vue?vue&type=script&lang=ts:326)
ModalCreateDapps.vue?77fa:227 _onFailed CodeSubmittableResult {dispatchError: Type, dispatchInfo: Map(3), internalError: undefined, events: Array(1), status: Type, …}blueprint: undefinedcontract: undefineddispatchError: Type {registry: TypeRegistry, createdAtHash: undefined, …}dispatchInfo: Map(3) {"weight" => Type, "class" => Type, "paysFee" => Type, …}events: [Map(3)]internalError: undefinedstatus: Type {registry: TypeRegistry, createdAtHash: undefined, …}isCompleted: (...)isError: (...)isFinalized: (...)isInBlock: (...)isWarning: (...)__proto__: SubmittableResult
_onFailed @ ModalCreateDapps.vue?77fa:227
...
```

```
ModalCreateDapps.vue?77fa:318 Uncaught TypeError: Cannot read property 'value' of undefined
    at eval (webpack-internal:///./node_modules/cache-loader/dist/cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/@vue/cli-plugin-typescript/node_modules/ts-loader/index.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader-v16/dist/index.js?!./src/components/dapps/ModalCreateDapps.vue?vue&type=script&lang=ts:326)
ModalCreateDapps.vue?77fa:227 _onFailed CodeSubmittableResult {dispatchError: Type, dispatchInfo: Map(3), internalError: undefined, events: Array(1), status: Type, …}blueprint: undefinedcontract: undefineddispatchError: Type {registry: TypeRegistry, createdAtHash: undefined, …}dispatchInfo: Map(3) {"weight" => Type, "class" => Type, "paysFee" => Type, …}events: [Map(3)]internalError: undefinedstatus: Type {registry: TypeRegistry, createdAtHash: undefined, …}isCompleted: (...)isError: (...)isFinalized: (...)isInBlock: (...)isWarning: (...)__proto__: SubmittableResult
_onFailed @ ModalCreateDapps.vue?77fa:227
...
```

Reply:
it seems another issue. because the file drop size is small, the file is not attached properly. will fix that. and I'll create another ticket for endowment estimation issue